pages/integrations/sso.md.erb

```diff
@@ -89,3 +89,17 @@ No, SSO providers are setup using a unique identifier and are unaffected when a
 
 ### Can I merge two organizations that use different SSO providers?  
 In short, yes, you can. However, merging Buildkite organizations that already have SSO providers might be a tricky scenario, and it's highly recommended that you contact support@buildkite.com for help or guidance before you attempt such migration.  
+
+### Why am I being asked for my password in the "Authorization Required" screen when signing in using SSO?  
+In short, signing in to your Buildkite organization requires both authorization and authentication procedures via Buildkite and your SSO provider. The two-factor authentication confirms that you are who you claim to be. Authorization (your password) verifies that you have permission to access and perform various actions in this Buildkite organization.  
```

Comment:
This doesn't quite match my understanding, which is more along the lines of https://www.okta.com/identity-101/authentication-vs-authorization/

![image](https://user-images.githubusercontent.com/95874/118995763-b139a280-b987-11eb-89f2-0ac14739b84d.png)

All of the password / SSO is authentication, and the authorization is the permission level of the account (in the context of each organization). So this **Authorization (your password) verifies that you have permission to access and perform various actions in this Buildkite organization.** feels a little off.

Reply:
I changed the definition to a less wordy one. I guess the "various actions" came from my need to elaborate "read + write" access permissions here. But if you ask me why, I'll draw a blank 😅.

Reply:
Yeah, but what I'm saying is that password is (imo) Authentication not Authorization.

So, to be super picky, the password verifies that that you are the person who owns the account that has the permissions? Does that sound right?